documentation/JsonSchema/usage.rst

```diff
@@ -25,7 +24,10 @@ You will have to do this::
 
     <?php
 
-    $normalizers = Vendor\Library\Generated\Normalizer\NormalizerFactory::create();
+    $normalizers = [
+        new Symfony\Component\Serializer\Normalizer\ArrayDenormalizer(),
+        new Vendor\Library\Generated\Normalizer\LazyNormalizer(),
```

Comment:
This name looks strange to me. Why not "JaneModelNormalizer"? The name should describe what is normalized, not how.